docs/sphinx/developers/file-reader.txt

```diff
@@ -79,6 +79,15 @@ table below together with the appropriate accessor method:
 
 All file formats are guaranteed to accurately report core metadata.
 
+Bio-Formats also converts and stores additional information which can be stored and retrieved 
+from the OME Metadata. These fields can be accessed in a similar way to the core metadata above.
+An example of such values would be the physical size of dimensions X, Y and Z. The accessor methods 
+for these properties return a :javadoc:`Length <ome/units/quantity/Length.html>` object which 
+contains both the value and unit of the dimension. These lengths can also be converted to other units using 
+:javadoc:`value(ome.units.unit.Unit) <ome/units/quantity/Length.html#value(ome.units.unit.Unit)>`
+An example of reading and converting these physical sizes values can be found in 
+:download:`ReadPhysicalSize.java <ReadPhysicalSize.java>`
```

Comment:
+1 Once we have defined a location for documenting the units system (see https://trello.com/c/FvyQj7OP/144-document-unit-api) we might also introduce a reference to the relevant page.